tool_path_planner/include/tool_path_planner/tool_path_planner_base.h

```diff
@@ -30,6 +30,7 @@ namespace tool_path_planner
     double tool_offset;               /** how far off the surface the tool needs to be */
     double intersecting_plane_height; /** Used in creating planes from that originate from an adjacent raster line to
                                           the surface*/
+    int nearest_neighbors;            /** how many neighbors are used to compute local normals*/
```

Comment:
I have not looked at this closely enough to know if it makes sense for these parameters to be added to the ProcessTool. However we should probably make these names specific enough that we know what they are. eg. simulator_tool_radius, etc.